python/labours/labours.py

```diff
@@ -1286,12 +1286,11 @@ def show_sentiment_stats(args, name, resample, start_date, data):
     deploy_plot(title, args.output, args.background)
 
 
-def show_devs(args, name, start_date, end_date, people, days):
+def show_devs(args, name, start_date, end_date, people, days, max_people=50):
     from scipy.signal import convolve, slepian
 
-    max_people = 50
     if len(people) > max_people:
-        print("Picking top 100 developers by commit count")
+        print("Picking top {} developers by commit count".format(max_people))
```

Comment:
```suggestion
        print("Picking top %s developers by commit count" % max_people)
```
Following p.5 of https://github.com/src-d/guide/blob/master/engineering/conventions/python.md#code-style

Reply:
Thank you for pointing it out! Addressed, history re-written.